src/clients/save_network.rb

```diff
@@ -70,6 +77,7 @@ def adjust_for_network_disks(file)
       )
 
       log.info("Network based device: #{remote_access}")
+=end
```

Comment:
Yes. It's temporarily commented (during the migration from storage to storage-ng) and we will need to replace that code with storage-ng enabled one in the future. That's why we are commenting all the code with this system. See https://github.com/yast/yast-packager/pull/177/files/0c62bfeb3c0ae9cc036d5e49f83aab21d3505cca#r74195595

https://github.com/yast/yast-storage-ng/blob/master/doc/installer-hacks.md